src/utils/wallet.js

```diff
@@ -369,14 +380,63 @@ const wallet = {
           continue;
         }
       }
+      unknownTokens.push(tokenData);
+    }
 
-      unknownTokens.push({
-        uid: tokenUid,
+    return unknownTokens;
+  },
+
+  /**
+   * Filters only the registered tokens from the allTokens list.
+   * Optionally filters only those with non-zero balance.
+   *
+   * @param params
+   * @param {Object[]} [params.allTokens] list of all available tokens
+   * @param {Object[]} [params.registeredTokens] list of registered tokens
+   * @param {Object[]} [params.tokensBalance] data about token balances
+   * @param {boolean} [params.hideZeroBalance] If true, omits tokens with zero balance
+   * @returns {object[]}
+   */
+  fetchRegisteredTokens(params = {}) {
+    const {allTokens, registeredTokens, tokensBalance, hideZeroBalance} = params;
```

Comment:
If those params are not optional, why don't we use actual parameters instead of an object?

Reply:
No, I think it is fine to keep it the way it as other methods on the same file are also using this style, I agree that it is a personal preference

Reply:
It took very little effort to change this to the current code style: it's better to keep the standards.
Refactored on [17d183e](https://github.com/HathorNetwork/hathor-wallet/pull/310/commits/17d183e585c0f8adc514086b8ecda940e497e16a).